### ISSUE TITLE
Why is there no corresponding answer in the test set?

### ISSUE THREAD

**Issue by FadedCosine** [JourneyDB/JourneyDB#4]
Dear Author, I have download the test set. But when I check the "test_questions.jsonl", I find there is no corresponding answer for all the tasks. I wonder if you have intentionally leave out this part, or if I can download the corresponding answers of the test set from somewhere else.

**Comment by FadedCosine** [comment]
I have one more question. Where can I find the 344 categories of style prompt? There are some categories in Figure 3, but I wonder if you cound provide the whole 344 categories.

**Comment by KeqiangSun** [comment]
> Dear Author, I have download the test set. But when I check the "test_questions.jsonl", I find there is no corresponding answer for all the tasks. I wonder if you have intentionally leave out this part, or if I can download the corresponding answers of the test set from somewhere else.

Thanks for your interest! Yes, we left out this part intentionally, since we are currently planning to organize a challenge on the generative visual understanding tasks. BTW, we have provided the annotation for the training and validation set. Please also check if you need it.

**Comment by KeqiangSun** [comment]
> I have one more question. Where can I find the 344 categories of style prompt? There are some categories in Figure 3, but I wonder if you cound provide the whole 344 categories.

We have further refined the style categories, which are better organized and will be released soon. Stay tuned!

**Comment by FadedCosine** [comment]
> 

Thanks for your kind reply.
